website/docs/sql_ddl.md

```diff
@@ -582,6 +597,28 @@ SHOW PARTITIONS hudi_table;
 --Drop partition：
 ALTER TABLE hudi_table DROP PARTITION (dt='2021-12-09', hh='10');
 ```
+
+### Show and drop index
+
+**Syntax**
+
+```sql
+-- Show Indexes
+SHOW INDEXES FROM tableIdentifier;
```

Comment:
please use consistent case (upper, lower, camel) as rest of docs

Reply:
Addressed